src/utils/mysql.utils.js

```diff
@@ -0,0 +1,19 @@
+const { format } = require("date-fns");
+
+function createPlaceholderString(count) {
+    // '?, ?, ' remove last 2 characters (', ')
+    // will return '?, ?'
+    return '?, '.repeat(count).slice(0, -2);
```

Comment:
Can explain on this line on code?
what is return from the .repeat(count).slice(0, -2)?

Reply:
`'?, '.repeat(5)` --> `"?, ?, ?, ?, ?, "`
`'?, '.repeat(5).slice(0, -2)` --> `"?, ?, ?, ?, ?"`